pkg/flux/flux.go

```diff
@@ -37,32 +35,25 @@ func (c *Client) PreFlight() error {
 		return errors.New("flux not found, required")
 	}
 
-	args := []string{"check", "--pre"}
+	if err := c.checkFluxVersion(); err != nil {
+		return err
+	}
 
-	if c.opts.Kubeconfig != "" {
-		args = append(args, "--kubeconfig", c.opts.Kubeconfig)
+	args := []string{"check", "--pre"}
+	for k, v := range c.opts.Flags {
+		if k == "kubeconfig" {
+			args = append(args, fmt.Sprintf("--%s", k), v)
+		}
```

Comment:
this fixes another issue with integration tests. units have been updated

Reply:
so only the kubeconfig flag gets passed to pre-flight? To we have to be wary about any other flags?

Reply:
:+1: sounds reasonable